### PR TITLE
CIV-637: :sparkles:: task to export all subjective responses

### PR DESCRIPTION
**Pull request by shrudhinvraj**
## JIRA Ticket

[CIV-637](https://commutatus.atlassian.net/browse/CIV-637)

## What?

task to send all user responses in a single file export

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -448,4 +448,45 @@ def verify_email_after_120_hours(user_id)
 		end
 	end
 
+	def export_all_subjective_responses(email)
+		consultation_list = Consultation.includes(:responses).select { |con| con.response_rounds.last.questions.count == 0}
```
Comment (manikandan0603) [inline app/mailers/user_mailer.rb:452]: @shrudhinvraj  - How many consultations will be there?

Comment (manikandan0603) [reply app/mailers/user_mailer.rb:452]: Do you have the count?

Comment (shrudhinvraj) [reply app/mailers/user_mailer.rb:452]: @manikandan0603  there will be 313 consultations which have subjective responses

**Comment by manikandan0603** [comment]
@shrudhinvraj  - Is this a one-time thing?

**Review by manikandan0603 — APPROVED** [review]

**Comment by shrudhinvraj** [comment]
> @shrudhinvraj - Is this a one-time thing?

@manikandan0603  yes they asked as to do as a onetime thing

**Pull request merged**